app/src/main/java/ch/epfl/sdp/appart/database/MockDatabaseService.java

```diff
@@ -80,6 +80,12 @@ public MockDatabaseService() {
         return result;
     }
 
+    @NonNull
+    @Override
+    public CompletableFuture<List<Card>> getCardsFilter(@NonNull String location) {
```

Comment:
You can quickly add a test for this new part of code

Reply:
Okay, I added some test now the branch coverage is above 80% and the overall coverage it has increased a little.